modules/enabling-isci-assisted-installer.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-iscsi-assisted-installer_{context}"]
+= Enabling iSCSI for the root disk in {op-system}
```

Comment:
What is {op-system}?  Is this section in the context of OCI?

Reply:
it's RHCOS

Reply:
Is this section in the context of OCI?  I don't know how this module fits into the rest of the docs.

Reply:
@avishayt yes, this is OCI only since it's only included in the upcoming OCI docs

Reply:
@faermanj : would you be OK to chime in on where this content could sit in our OCI documentation?